src/main/resources/reference.conf

```diff
@@ -148,7 +148,11 @@ cassandra-journal {
   # The id for the local datacenter of the Cassandra hosts it should connect to. 
   # By default, this property is not set resulting in Datastax's standard round robin policy being used.
   local-datacenter = ""
-  
+
+  # Number of hosts from non-local datacenter to use as a fall-back policy.
+  # Works only when local-datacenter is set
+  used-hosts-per-remote-dc = 0
```

Comment:
does this change anything related to default behavior, i.e. when not setting `withUsedHostsPerRemoteDc`?